python/setup.py

```diff
@@ -18,12 +19,14 @@
 
 MODEL_DIR = "stan"
 MODEL_TARGET_DIR = os.path.join("prophet", "stan_model")
-
+MODEL_GPU = False
+MODEL_MULTI_CORES = False
 CMDSTAN_VERSION = "2.26.1"
 BINARIES_DIR = "bin"
 BINARIES = ["diagnose", "print", "stanc", "stansummary"]
 TBB_PARENT = "stan/lib/stan_math/lib"
 TBB_DIRS = ["tbb", "tbb_2019_U8"]
+OPEN_CL = "-L"+"\"C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8/lib/Win32/OpenCL.lib\""
```

Comment:
You should not use absolute path, instead use generic:
**Windows:**
`OPEN_CL = "-L\"" + "%CUDA_PATH%/lib/Win32/OpenCL.lib\""`
**Linux:**
`OPEN_CL = "-L\"" + "$CUDA_PATH/lib64/OpenCL.lib\""`

Note: **%CUDA_PATH%** is an environment variable that points to the CUDA installation directory.

Reply:
Good idea! I'll work on adding that with a checker for the OS

Reply:
Awesome, just added